packages/odyssey-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -115,8 +112,7 @@ export const Checkbox = withTheme(
               {indeterminate ? <SubtractIcon /> : <CheckIcon />}
             </span>
           </span>
-
-          {label}
+          <span id={ariaLabelledById}>{label}</span>
         </label>
         {error && <Field.Error id={oid}>{error}</Field.Error>}
```

Comment:
Wondering if we need an `aria-live="polite"` attribute here to announce form field errors to screen readers.  Thoughts, @jeffbelser-okta , @andrewberg-okta ?

Reply:
Or perhaps we should use `aria-errormessage` (along with `aria-live`) per https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-errormessage